components/CollectiveModal.tsx

```diff
@@ -18,7 +18,23 @@ export const collectiveQuery = gql`
       slug
       createdAt
       description
-      updates(limit: 3) {
+      admins: members(role: [ADMIN], limit: 0) {
+        totalCount
+      }
+      ... on Collective {
+        host {
+          name
+          slug
+        }
+      }
+      ... on Fund {
+        host {
+          name
+          slug
+        }
+      }
```

Comment:
I've tried this but it is not returning anything with this syntax

Reply:
Fixed this issue now by adding AccountWithHost in Apollo config.

Also included the host in the static props instead of fetching it in the modal (we're already getting it in the fetch-data script), for a 7% increase in static data size.